tests/test_filtsmooth/test_gaussfiltsmooth/filtsmooth_testcases.py

```diff
@@ -149,8 +151,14 @@ def dh(t, x):
     r = var * np.eye(1)
     initmean = np.ones(2)
     initcov = var * np.eye(2)
-    dynamod = pnfs.statespace.DiscreteGaussian(f, lambda t: q, df)
-    measmod = pnfs.statespace.DiscreteGaussian(h, lambda t: r, dh)
+    dynamod = pnfs.statespace.DiscreteGaussian(
+        2,
+        2,
+        f,
+        lambda t: q,
+        df,
```

Comment:
Does this get nicer if you delete the final comma?

Reply:
:D 

Not sure, but it does if I add keywords to the arguments.